src/classes/RD2_RecurringDonations_TEST.cls

```diff
@@ -162,6 +163,40 @@ private with sharing class RD2_RecurringDonations_TEST {
             'The triggerOld parameter should contain the data from the oldList parameter');
     }
 
+    /***
+    * @description Verifies schedule is created when recurring donation is inserted
+    */
+    @isTest
+    private static void shouldCreateNewRDScheduleOnInsert() {
+        RD2_EnablementService_TEST.setRecurringDonations2Enabled();
+
+        Contact contact = getContact();
+
+        List<npe03__Recurring_Donation__c> rds = new List<npe03__Recurring_Donation__c>{
+                new TEST_RecurringDonationBuilder()
```

Comment:
This builder class supports both legacy and RD2 format RD's. To know exactly which format you're creating, I recommend using the following format.
```suggestion
                TEST_RecurringDonationBuilder.constructEnhancedBuilder()
```